docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
     environment:
       MYSQL_DATABASE: canopeum_db
       MYSQL_HOST_CANOPEUM: /run/secrets/MYSQL_HOST_CANOPEUM
-      MYSQL_PORT_CANOPEUM: /run/secrets/MYSQL_PORT_CANOPEUM
+      MYSQL_PORT_CANOPEUM: 3306
```

Comment:
This secret is also found at: https://github.com/BesLogic/releaf-canopeum/blob/e2aba364fbe8b57b3f0728128cc5ee4580b2ef8b/canopeum_backend/canopeum_backend/settings.py#L186
Are we ok always using the default hardcoded value and letting it be configurable only locally ? (it's already what's happening if we weren't providing the secrets, I just wanna make sure )

I'm also never sure if this is the internal or external port...

Reply:
Verified! It is the fix I had to do to support the connection with the db